conf/db/migration/metrics_portal_ddl/common/V11__change_to_query_alerts.sql

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017 InscopeMetrics
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ALTER TABLE portal.alerts ADD query text;
+
+UPDATE portal.alerts set query = 'from 10 minutes ago to now select ' + metric + ' where scope = ''host'' and cluster = ''' + cluster + ''' and '
```

Comment:
These look like MQL.

Reply:
Yeah.... what do you think I should do with the existing alerts?

Reply:
I don't think we should do data migration inline. MV bump, new tablespace/names/whatever, and instructions to migrate data.